Changelog.md

```diff
@@ -4,6 +4,10 @@ Language Features:
 
 
 Compiler Features:
+ * Commandline Interface: Add ``--ast-compact-json`` option with Yul input (e.g. with ``--strict-assembly``, ``--yul``).
+ * Commandline Interface: Add ``--ir-ast-json`` and ``--ir-optimized-ast-json`` output options which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
```

Comment:
I think that the removal of the suffix was meant to apply to CLI options too. The `-json` here is just as superfluous.

```suggestion
 * Commandline Interface: Add ``--ir-ast`` and ``--ir-optimized-ast`` output options which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
```

Reply:
It's consistent with other options, so let's keep it as is.